app/models/miq_cockpit_ws_worker/authenticator.rb

```diff
@@ -20,14 +20,14 @@ def self.ssh_command
 
   def creds_for_vm(vm)
     return nil unless vm
-    creds = vm.respond_to?(:key_pairs) ? vm.key_pairs.first : nil unless creds
-    creds ? creds : Authentication.new
+    creds ||= vm.key_pairs.first if vm.respond_to?(:key_pairs)
```

Comment:
don't think you need `||=` for the first assigment to a variable.  `=` should be sufficient here.

Reply:
the previous one had `unless creds`, so I think that is why the `||=` was added

although, it is a little wierd, not sure what `creds` is being read in the new version?

Reply:
removed